src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Gripper extends SubsystemBase {
+    private static Gripper INSTANCE;
+    private final GripperInputs inputs = new GripperInputs();
+    private final GripperIO io;
+    private final Mechanism2d mechanism2d = new Mechanism2d(0, 0);
+    private final MechanismRoot2d root = mechanism2d.getRoot("Gripper", 0, 0);
+    private final MechanismLigament2d shoulder =
+            root.append(new MechanismLigament2d("Gripper", 0, 0));
+    private final GripperInputsAutoLogged input = new GripperInputsAutoLogged();
```

Comment:
Like in the elevator, this should be made and taken from the `GripperIO`.

Reply:
8cb9a19153fb50cc13351f1d45f0860b4e10e42c